src/app/children/health-checkup/health-checkup.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { HealthCheckupComponent } from './health-checkup.component';
+
+describe('HealthCheckupComponent', () => {
+  let component: HealthCheckupComponent;
+  let fixture: ComponentFixture<HealthCheckupComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ HealthCheckupComponent ]
```

Comment:
I think you are missing some imports here, that's why the test fails.

Reply:
It took some time but I think I managed to fix this